docs/website/docs/getting-started/index.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started Guide
+
+## Setup
+Setup your local environment by installing IREE's prebuilt binaries. See the comprehensive
+installation [guide](../bindings/python.md) for installation of individual components, the default installation is below.
```

Comment:
"Use the following command for the default installation, or check out the comprehensive installation guide if your needs are more complex.

Reply:
Done.